packages/autorest.python/test/vanilla/legacy/AcceptanceTests/test_constants.py

```diff
@@ -46,3 +47,12 @@ def test_put_client_constants_override():
         assert client._config.header_constant == False
         assert client._config.query_constant == 0
         assert client._config.path_constant == "new_path"
+
+def test_models():
+    models.ModelAsStringNoRequiredOneValueDefault()
```

Comment:
what does this test do? Should we check the defaults are in the properties?

Reply:
If we don't `support x-ms-client-default`, the test will raise `TypeError(missing 1 required postional argument)`. But anyway, I will add the check for default value.